recipes/newrelic/infrastructure/logs/windows-logs.yml

```diff
@@ -37,9 +37,14 @@ install:
 
     setup:
       cmds:
-        - powershell -command '$logs = "C:\\Program Files\\New Relic\\newrelic-infra\\logging.d\\logs.yml"; if (Test-Path $logs) { Remove-Item $logs }'
-        - powershell -command 'echo "{{.LOGS_CONTENT}}" >> "C:\\Program Files\\New Relic\\newrelic-infra\\logging.d\\logs.yml"'
-        - powershell -command 'Restart-Service -Name newrelic-infra'
+        - |
+          powershell -command '
+          $logs = "C:\\Program Files\\New Relic\\newrelic-infra\\logging.d\\logs.yml"
+          if (-not (Test-Path $logs)) {
```

Comment:
@noahmmcgivern why do we now have a "-not" in the condition? We didn't have this originally.
The logs.yml file is generated by this recipe, so it should be ok to remove with every install.